### PR TITLE
New report scanners, clair, hadolint and dockerlint

### PR DESCRIPTION
**Pull request by amandel**
Hope I got all correct.

### CODE REVIEW

File: plugin/src/main/webapp/icons/LICENSE.txt [hunk 1]
```diff
@@ -169,4 +169,12 @@ https://github.com/junit-team/junit5/issues/2165#issuecomment-578806575
...
 The 24x24 and 48x48 icons were created from this file. No modifications were made, except
-the scale that was changed.
\ No newline at end of file
+the scale that was changed.
+
+------------------------------------------------------------------------------
+
+Hadolint icons:
+
+Taken from https://hadolint.github.io/hadolint/img/cat_container.png and rescaled.
```
Comment (uhafner) [inline plugin/src/main/webapp/icons/LICENSE.txt:178]: Does Hadolint have a mailing list or chat? Can you please ask for a permission to reuse the icon? Icons are sometimes problematic to reuse.

Comment (amandel) [reply plugin/src/main/webapp/icons/LICENSE.txt:178]: Sure I'll do so.

Comment (amandel) [reply plugin/src/main/webapp/icons/LICENSE.txt:178]: I now have feedback at https://github.com/hadolint/hadolint/issues/449#issuecomment-658594755 - is sufficient from your point of view?

Comment (uhafner) [reply plugin/src/main/webapp/icons/LICENSE.txt:178]: Yes, that is fine now!

**Comment by amandel** [comment]
Thanks for releasing!

**Pull request merged**